repo/blob/filesystem/filesystem_storage_capacity_windows.go

```diff
@@ -0,0 +1,48 @@
+//go:build windows
+// +build windows
+
+package filesystem
+
+import (
+	"context"
+	"syscall"
+	"unsafe"
+
+	"github.com/pkg/errors"
+
+	"github.com/kopia/kopia/repo/blob"
+)
+
+// nolint:gosec
+func (fs *fsStorage) GetCapacity(ctx context.Context) (blob.Capacity, error) {
+	dll, err := syscall.LoadLibrary("kernel32.dll")
+	defer syscall.FreeLibrary(dll) // nolint:errcheck
+
+	if err != nil {
+		return blob.Capacity{}, errors.Wrap(err, "load kernel32 library")
+	}
+
+	proc, err := syscall.GetProcAddress(dll, "GetDiskFreeSpaceExW")
```

Comment:
According to 

https://stackoverflow.com/questions/54664751/how-to-get-total-size-of-a-drive-in-windows

there's a simpler solution where you import  "golang.org/x/sys/windows", can we use it here?